src/timeline/TimelineViewManager.cpp

```diff
@@ -339,6 +339,7 @@ TimelineViewManager::toggleCameraView()
 void
 TimelineViewManager::openImageOverlay(QString mxcUrl, QString eventId) const
 {
+        if (mxcUrl.isEmpty() || mxcUrl.isNull()) { return; }
```

Comment:
```suggestion
        if (mxcUrl.isEmpty()) { return; }
```

isEmpty returns `true`, if the string is `null`.

![grafik](https://user-images.githubusercontent.com/16312324/100644636-7db08080-3333-11eb-953b-51abf8bd9bab.png)